addons/l10n_it_edi/models/account_chart_template.py

```diff
@@ -16,4 +16,11 @@ def _load(self, company):
         if template == self.env.ref('l10n_it.l10n_it_chart_template_generic'):
             company.account_sale_tax_id = self.env.ref(f'l10n_it.{company.id}_22v')
             company.account_purchase_tax_id = self.env.ref(f'l10n_it.{company.id}_22am')
+            tax = self.env.ref(f'l10n_it.{company.id}_00eu', raise_if_not_found=False)
+            if tax:
+                tax.write({
+                    'l10n_it_has_exoneration': True,
+                    'l10n_it_kind_exoneration': 'N3.2',
+                    'l10n_it_law_reference': 'Art. 41, DL n. 331/93',
+                })
```

Comment:
I think it's fine like this (more explicit), but just so you know, write in an empty recordset doesn't do anything, it doesn't raise an error or anything, it just `return True` without doing anything